archivy/cli.py

```diff
@@ -47,15 +47,21 @@ def init(ctx):
         default=str(Path(".").resolve()),
     )
 
-    es_enabled = click.confirm(
-        "Would you like to enable Elasticsearch? For this to work "
-        "when you run archivy, you must have ES installed."
-        "See https://archivy.github.io/setup-search/ for more info."
+    desires_search = click.confirm(
+        "Would you like to have search on your knowledge base contents?"
```

Comment:
Can we better phrase this? It looks like something is missing in this line?

Perhaps 'indexing' is the right word, or a ',' is missing?

Reply:
* Would you like to have indexing on your know....
* "Would you like to have 'search' on your knowledge base contents?"
* etc etc, does it sound right? I am not sure either...

Reply:
Hmm yeah you're right, it could be improved. What would you propose?

Reply:
- "Would you like to enable search on your knowledge base contents?"
Maybe that could work?

Reply:
> * "Would you like to enable search on your knowledge base contents?"

Yea, this looks perfect.